src/components/Empty/Empty.tsx

```diff
@@ -77,7 +78,11 @@ export const Empty: FC<EmptyProps> = React.forwardRef(
         </div>
         {title && <div className={styles.emptyTitle}>{title}</div>}
         {description && (
-          <div className={styles.emptyDescription}>{description}</div>
+          <div
+            className={`${styles.emptyDescription} ${descriptionClassNames}`}
```

Comment:
nit: Rather than a templated string, you could use mergeClasses.

Reply:
Done